app/serializers/api/enterprise_shopfront_serializer.rb

```diff
@@ -63,7 +63,7 @@ def producers
 
     def hubs
       ActiveModel::ArraySerializer.new(
-        enterprise.distributors, each_serializer: Api::EnterpriseThinSerializer
+        enterprise.distributors.not_hidden, each_serializer: Api::EnterpriseThinSerializer
```

Comment:
This looks like it would work. Can you add a spec for it? You can add to `spec/serializers/api/enterprise_shopfront_serializer_spec.rb`. You would need to write a spec that fails without this change but then passes with this change. It needs to verify the absence of a hidden distributor.

Reply:
Sure @mkllnk

Reply:
Done @mkllnk Please re review